cfpb/cfgov-refresh
# Update the FCM to CFv4 markup and styles

The project is currently using a really old version of the Featured Content Module. Oddly, it was never updated to use the atomic version within this code base. This change updates the markup to the atomic pattern and duplicates the Less from the latest version in Capital Framework. The /organisms/featured-content-module.less can be removed when this project is updated to use CFv4.

## Removals

- Removed old FCM enhancements

## Changes

- Updated the atomic FCM styles to the current code from Capital Framework
- Updated the FCM markup to match

## Testing

- Run the site
- Run `gulp build`, 
- Check http://localhost:8000/about-us/the-bureau/ and ensure the black bar is gone

## Screenshots

__Before:__

![screen shot 2017-05-24 at 3 35 36 pm](https://cloud.githubusercontent.com/assets/1280430/26424388/a72ae9fc-4096-11e7-9fac-ddf44cadedbe.png)

__After:__

![screen shot 2017-05-24 at 2 04 35 pm](https://cloud.githubusercontent.com/assets/1280430/26424370/9b1f5b66-4096-11e7-9c4d-0d1343554970.png)

__Before:__

![screen shot 2017-05-24 at 3 36 34 pm](https://cloud.githubusercontent.com/assets/1280430/26424428/cbcc44c2-4096-11e7-92e0-846a44a6fe78.png)

__After:__ (I don't have the /f/ dir locally so I replaced the image with cats)

![screen shot 2017-05-24 at 2 04 44 pm](https://cloud.githubusercontent.com/assets/1280430/26424460/e693390a-4096-11e7-8405-3bf3981b9df6.png)

## Notes

- This PR is two 🐦  with one 🗿 , fixing the current visual bug on cf.gov and updating the project to save the CF update team the work of doing it later.

## Checklist
